.golangci.yml

```diff
@@ -53,6 +53,10 @@ issues:
     - linters:
         - revive
       text: "redefines-builtin-id"
+    # FIXME temporarily suppress these. See https://github.com/containerd/containerd/pull/10250
+    - text: "SA1019: grpc.(WithBlock|FailOnNonTempDialError|WithReturnConnectionError|DialContext|Dial) is deprecated"
+      linters:
+        - staticcheck
```

Comment:
I added this temporary exclude-rule for staticcheck because this issue is too large to handle in this PR. This is caused by grpc's deprecation ([failed lint job](https://github.com/containerd/containerd/actions/runs/9967129116/job/27540310960?pr=10410)).

https://github.com/containerd/containerd/pull/10250 is fixing this deprecation.

Reply:
#10250 looks like ready to be merged so I believe this exception won't be needed.

Reply:
Thanks! I will update my PR once the pr is merged!!

Reply:
removed this excluding rule. Thanks!!